### ISSUE TITLE
Documentation page linked from github page contains no documentation

### ISSUE THREAD

**Issue by WolframKahl** [AmpersandTarski/Ampersand#572]
At http://ampersandtarski.gitbooks.io/documentation/ , the only useful thing is `About Ampersand`, but there is nothing else there. (Following the `Tools we Use`  link however gets me to a functioning gitbook with non-empty table of contents to the left.)

**Comment by hanjoosten** [comment]
I agree that that page is somewhat confusing. However, did you see the `Read` button in the upper right corner? 

![image](https://cloud.githubusercontent.com/assets/1159672/20784345/f30a4254-b799-11e6-9e11-3a059b833f47.png)

This page currently gives the oppertunity to read the page in several formats (pdf, html etc. ). and is generated by gitbook. Nothing much we can do about it unless we are willing to spend a lot of effort. 

How did you get on this page? Did you use ampersandtarski.github.io ? Would it help to change the link over there to be the same one as is behind the green `Read` button?

**Comment by WolframKahl** [comment]
The `Read` button gets me to exactly the same content (that is, only: About Ampersand) on a prettier page.

I got to that page from https://github.com/AmpersandTarski/Ampersand , which I believe is currently intended to be the main public entry page for Ampersand, and which links there in the first sentence under Documentation:
````
The best place to look at if you are new to Ampersand, and you want to install the software, is at our documentation. 
````

**Comment by WolframKahl** [comment]
I received an HTML-only e-mail today, and I have no idea why GitBook thinks that this is *my* book:
````
From: GitBook <contact@gitbook.com>
To: kahl@cas.mcmaster.ca
Subject: [Documentation of Ampersand] Update (master/814a3ea) has failed
Date: Thu, 01 Dec 2016 12:24:22 +0000


        [Documentation of Ampersand] Update (master/814a3ea) has failed
                                    [1][IMG]

   Update Failed
   The update [2](master/814a3ea) of your book
   [3]ampersandtarski/documentation has failed after 18.605 seconds.
   Access your update details to fix it and publish a new version.
   Having trouble? Check out the list of [4]common errors.

   [5]Update Logs

           © [6]GitBook 2016                [7]Notification Settings

   !DSPAM:5840167875381804284693!

References

   Visible links
   1. https://www.gitbook.com/
   2. https://www.gitbook.com/notification/58401675144fc90400475c2f?redirect=https%3A%2F%2Fwww.gitbook.com%2Fbook%2Fampersandtarski%2Fdocumentation
   3. https://www.gitbook.com/notification/58401675144fc90400475c2f
   4. https://help.gitbook.com/content/why-are-my-updates-failing.html
   5. https://www.gitbook.com/notification/58401675144fc90400475c2f?redirect=https%3A%2F%2Fwww.gitbook.com%2Fbook%2Fampersandtarski%2Fdocumentation
   6. https://www.gitbook.com/
   7. https://www.gitbook.com/settings/notifications
````

**Comment by WolframKahl** [comment]
I tried http://ampersandtarski.github.io — as an entry page, that does look a bit better, but the link to the documentation there still gives me no content beyond `About Ampersand`, also behind the Read button.

**Comment by WolframKahl** [comment]
I tried http://ampersandtarski.github.io — as an entry page, that does look a bit better.

The link to the documentation there still at first seemed to give me no content beyond `About Ampersand`, also behind the Read button. But ages later (accidentally discovered by returning to that tab), the table of contents appeared to the left. I tried it again: Way over ten seconds, and long afteer the last `Transferring data from ...` in the browser goes away.
